enaml/core/parser/base_parser.py

```diff
@@ -495,125 +496,104 @@ def p_pragma_arg3(self, p):
-        node = enaml_ast.AliasExpr()
-        node.lineno = p.lineno(1)
-        node.name = p[2]
-        node.target = p[4]
-        node.chain = tuple(p[5])
-        p[0] = node
-
-    # -------------------------------------------------------------------------
-    # ConstExpr
-    # -------------------------------------------------------------------------
-    def p_const_expr1(self, p):
-        ''' const_expr : CONST NAME EQUAL test NEWLINE '''
-        lineno = p.lineno(1)
-        body = p[4]
-        body.lineno = lineno
-        ast.fix_missing_locations(body)
-        expr = ast.Expression(body=body)
-        python = enaml_ast.PythonExpression(ast=expr, lineno=lineno)
-        node = enaml_ast.ConstExpr()
-        node.lineno = lineno
-        node.name = p[2]
-        node.expr = python
-        p[0] = node
-
-    def p_const_expr2(self, p):
-        ''' const_expr : CONST NAME COLON dotted_name EQUAL test NEWLINE '''
-        lineno = p.lineno(1)
-        body = p[6]
-        body.lineno = lineno
-        ast.fix_missing_locations(body)
-        expr = ast.Expression(body=body)
-        python = enaml_ast.PythonExpression(ast=expr, lineno=lineno)
-        node = enaml_ast.ConstExpr()
-        node.lineno = lineno
-        node.name = p[2]
-        typename = ast_for_dotted_name(p[4])
-        typename.lineno = lineno
-        node.typename = typename
-        node.expr = python
-        p[0] = node
-
-    # -------------------------------------------------------------------------
-    # StorageExpr
+    # StorageExpr | AliasExpr | ConstExpr
     # -------------------------------------------------------------------------
     def _validate_storage_expr(self, kind, lineno, lexer):
         if kind not in ('attr', 'event'):
             syntax_error('invalid syntax', FakeToken(lexer, lineno))
 
-    def p_storage_expr1(self, p):
-        ''' storage_expr : NAME NAME NEWLINE '''
+    def p_storage_alias_const_expr1(self, p):
+        ''' storage_alias_const_expr : NAME NAME NEWLINE '''
         kind = p[1]
         lineno = p.lineno(1)
-        self._validate_storage_expr(kind, lineno, p.lexer.lexer)
-        node = enaml_ast.StorageExpr()
-        node.lineno = lineno
-        node.kind = kind
-        node.name = p[2]
-        p[0] = node
-
-    def p_storage_expr2(self, p):
-        ''' storage_expr : NAME NAME COLON dotted_name NEWLINE '''
+        # Alias expression
+        if kind == "alias":
+            node = enaml_ast.AliasExpr()
+            node.lineno = lineno
+            node.name = p[2]
+            node.target = p[2]
+            p[0] = node
+        # Storage expression
+        else:
+            self._validate_storage_expr(kind, lineno, p.lexer.lexer)
+            node = enaml_ast.StorageExpr()
+            node.lineno = lineno
+            node.kind = kind
+            node.name = p[2]
+            p[0] = node
+
+    def p_sstorage_alias_const_expr2(self, p):
```

Comment:
extra s in storage

Reply:
Good catch and proof we need more tests !!!